src/components/td/td-styles.jsx

```diff
@@ -20,12 +20,61 @@ export default theme => {
       minWidth: 20,
       paddingTop: 8,
       paddingBottom: 8,
+      display: 'block',
+
+      '&::before': {
+        display: 'block',
+        content: 'attr(data-title)',
+      },
 
       [mixins.media('md')]: {
         fontSize: 14,
+        display: 'inline-block',
         minWidth: 40,
         paddingTop: 10,
         paddingBottom: 10,
+        '&::before': {
+          content: 'none',
+        },
+      },
+
+      '&.flexOrder': {
+        order: props => props.flexOrder,
+
+        [mixins.media('md')]: {
+          order: () => 1,
+        },
+      },
+
+      '&.flexBasis': {
+        flexBasis: props =>
+          props.flexBasisMobile
+            ? `${props.flexBasisMobile}${typeof props.flexBasisMobile === 'number' ? '%' : ''}`
+            : `${props.flexBasisDesktop}${typeof props.flexBasisDesktop === 'number' ? '%' : ''}`,
```

Comment:
Why assume percentages here? `flex-basis` can accept many value types and reading the prop `flexBasisMobile` doesn't indicate percentage. Why not pass in the percentage via the prop?

Reply:
Can you explain more ? If its a number its percent, if its string its pixels, just like all the others like this one all over the UI kit

Reply:
Captured as part of #347